datahub-web-react/src/app/ingest/source/builder/RecipeForm/common.tsx

```diff
@@ -257,19 +276,44 @@ export const INCLUDE_LINEAGE: RecipeField = {
 export const INCLUDE_TABLE_LINEAGE: RecipeField = {
     name: 'include_table_lineage',
     label: 'Include Table Lineage',
-    tooltip: 'Whether or not table lineage should be ingested.',
+    tooltip: 'Extract Tabel-Level lineage metadata. Enabling this may increase the duration of the extraction process.',
     type: FieldType.BOOLEAN,
     fieldPath: 'source.config.include_table_lineage',
     rules: null,
 };
 
-export const PROFILING_ENABLED: RecipeField = {
+const isProfilingEnabled = 'source.config.profiling.enabled';
+const isTableProfilingOnly = 'source.config.profiling.profile_table_level_only';
```

Comment:
what do you think of adding `...FieldPath` at the end of these variables?

Reply:
sure thing